modules/wallet/transactionbuilder.go

```diff
@@ -17,7 +108,7 @@ type transactionBuilder struct {
 	siacoinInputs []int
 	siafundInputs []int
 
-	wallet *Wallet
+	wallet walletSpender
```

Comment:
not that it's a huge deal, but the name probably shouldn't be 'wallet' since it's actually a walletSpender.